google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriter.java

```diff
@@ -485,17 +481,16 @@ private void onFailure(Throwable t) {
     private void onSuccess(AppendRowsResponse response) {
       for (int i = 0; i < inflightRequests.size(); i++) {
         AppendRowsResponse.Builder singleResponse = response.toBuilder();
-        // if (offsetList.get(i) > 0) {
-        //   singleResponse.setOffset(offsetList.get(i));
-        // } else {
-        //   long actualOffset = response.getOffset();
-        //   for (int j = 0; j < i; j++) {
-        //     actualOffset +=
-        //
-        // inflightRequests.get(j).message.getProtoRows().getRows().getSerializedRowsCount();
-        //   }
-        //  singleResponse.setOffset(actualOffset);
-        // }
+        if (response.getAppendResult().hasOffset()) {
+          long actualOffset = response.getAppendResult().getOffset().getValue();
+          for (int j = 0; j < i; j++) {
+            actualOffset +=
+                inflightRequests.get(j).message.getProtoRows().getRows().getSerializedRowsCount();
+          }
+          LOG.info("Acutal offset:" + actualOffset);
```

Comment:
Just a note that eventually we want to make these logger output `FINER` or `FINEST` since we don't want to overwhelm developers with granular information like this unless it is very important for them to know.

Reply:
Removed.